README.md

```diff
@@ -11,10 +11,7 @@ The Checkly CLI and Constructs in this repo together form the basic building blo
 This goal of this repo and the larger MaC project is to deliver a Javascript/Typescript-native workflow for creating,
 debugging, deploying and life cycling synthetic monitors (checks) at scale, from your code base.
 
-This project is now in `alpha`:
-1. Our goal is to first make a great DX / workflow for writing, debugging and deploying checks: focus on terminal, CLI and programming model.
-2. Not all TS types are done yet and also UI integration is largely not there.
-3. New check types, resources (like alerting channels) will follow soon.
+This project is now in `alpha`, we aim for `beta` mid-February '23. `GA` should be before end Q1 '23.
```

Comment:
Should we communicate a bit what users should expect from "alpha", "beta", and "GA"? Maybe something like "During the alpha phase there may be some bugs, missing features, and occasionally a breaking change."

Reply:
Should we have a  "Known limitations" section close to the `alpha` statement?

Reply:
@stefanjudis @clample let's add that later when we have more clarity around this. Might be next week already